### ISSUE TITLE
Detect when metadata emails are not processed

### ISSUE THREAD

**Issue by ryscher** [CDL-Dryad/dryad-product-roadmap#1657]
Rarely, something changes that prevents the metadata emails from processing. It's not a "terrible" bug, but currently, we only learn about this when someone complains, which may take weeks, so it ends up being a larger problem than if we caught it immediately.

**Comment by ahamelers** [comment]
See #3137

**Comment by ryscher** [comment]
Create an alert when metadata has not been received from a partner in over a month. (Using the same date mechanism as the "Date last receieved" on https://datadryad.org/journals)

**Comment by ryscher** [comment]
Ryan to discuss with Maria, the priority and process for this

**Comment by ryscher** [comment]
Yes, we still want to do this. Create a process that will check each journal, and alert us if we have not received metadata from that journal in the past month.
- For email integrations, use the same date calculation that is on the journals page for "Data last received"
- For API integrations, use the date we last updated a dataset's activity log with one of the messages indicating the API was used for that journal.
